test/helpers.go

```diff
@@ -0,0 +1,94 @@
+package test
```

Comment:
same here, I'd prefer not to limit tests based on the OS unless we absolutely have to.  And in those cases we should probably be using build tags with no-op implementations for other distros so they will still run cleanly.